Gemfile

```diff
@@ -99,6 +99,7 @@ instance_eval File.read local_gemfile if File.exist? local_gemfile
 
 group :test do
   gem "minitest-bisect"
+  gem "minitest-retry"
```

Comment:
The test group is not include in the tests O_O

Reply:
🤦‍♂️ I'll fix that.